src/common/language_dropdown.js

```diff
@@ -0,0 +1,14 @@
+import $ from 'jquery'
+
+export default class LanguageDropdown {
```

Comment:
In es6 encapsulation is the default behavior, therefore you don't need to define classes to ensure encapsulation. You could simply write:
```
export () => {
  $('ul#select_language li')
    .click(function onClick() {
...
}

```